protocol/testing/testnet-local/local.sh

```diff
@@ -95,7 +95,7 @@ create_validators() {
 		cat <<<"$new_file" >"$VAL_CONFIG_DIR"/node_key.json
 
 		edit_config "$VAL_CONFIG_DIR"
-		use_slinky "$VAL_CONFIG_DIR"
+		# use_slinky "$VAL_CONFIG_DIR"
```

Comment:
can we just delete this?

Reply:
We can if you want. My idea is to re-enable it in the followup PR which will add the separate slinky docker image to the localnet.